### PR TITLE
Added macros with alternative names; improved sizes and spacing with different kinds of brackets

### PR DESCRIPTION
**Pull request by ickc**
A continuation from #22.

This one should be clean from any development files.

### CODE REVIEW

File: physics/physics.js [hunk 1]
```diff
@@ -20,5 +20,4 @@
...
...,2],ket:["Macro","{	\\left\\vert { #1 } \\right\\rangle	}",1],bra:["Macro","{	\\left\\langle { #1} \\right\\vert	}",1],innerproduct:["Macro","{	\\left\\langle {#1} \\mid { #2} \\right\\rangle	}",2],braket:["Macro","{	\\left\\langle {#1} \\mid { #2} \\right\\rangle	}",2],outerproduct:["Macro","{	\\left\\vert { #1 } \\right\\rangle\\left\\langle { #2} \\right\\vert	}",2],dyad:["Macro","{	\\left\\vert { #1 } \\right\\rangle\\left\\langle { #2} \\right\\vert	}",2],ketbra:["Macro","{	\\left\\vert { #1 } \\right\\rangle\\left\\langle { #2} \\right\\vert	}",2],op:["Macro","{	\\left\\vert { #1 } \\right\\rangle\\left\\langle { #2} \\right\\vert	}",2],expectationvalue:["Macro","{	\\left\\langle {#1 } \\right\\rangle	}",1],expval:["Macro","{	\\left\\langle {#1 } \\right\\rangle	}",1],ev:["Macro","{	\\left\\langle {#1 } \\right\\rangle	}",1],matrixelement:["Macro","{	\\left\\langle{ #1 }\\right\\vert{ #2 }\\left\\vert{#3}\\right\\rangle	}",3],matrixel:["Macro","{	\\left\\langle{ #1 }\\right\\vert{ #2 }\\left\\vert{#3}\\right\\rangle	}",3],mel:["Macro","{	\\left\\langle{ #1 }\\right\\vert{ #2 }\\left\\vert{#3}\\right\\rangle	}",3]}})}),MathJax.Ajax.loadComplete("[Contrib]/physics/physics.js");
```
Comment (pkra) [inline physics/physics.js]: This seems unintentional?

File: physics/README.md [hunk 2]
```diff
@@ -1,218 +1,72 @@
+---
+HTML Header:	<script type="text/x-mathjax-config">MathJax.Ajax.config.path.Contrib="https://cdn.mathjax.org/mathjax/contrib",MathJax.Hub.Register.StartupHook("TeX Jax Ready",function(){MathJax.Hub.Insert(MathJax.InputJax.TeX.Definitions.macros,{cancel:["Extension","cancel"],bcancel:["Extension","cancel"],xcancel:["Extension","cancel"],cancelto:["Extension","cancel"]})}),MathJax.Hub.Config({TeX:{equationNumbers:{autoNumber:"AMS"},extensions:["[Contrib]/physics/physics.js","[Contrib]/siunitx/siunitx.js"]}});</script><script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML-full"></script>
```
Comment (pkra) [inline physics/README.md]: Just wanted to double check if you want this header to show up; GitHub rendering doesn't handle it too well but that's your call.

**Comment by pkra** [comment]
Thanks for this! I added two comments but otherwise this looks good.

**Comment by ickc** [comment]
Hi, I updated the readme.

For the physics.js, do you mean there are a lot of tab characters there? I used uglifyjs2 and thought it would handled them. I cleaned them up just now.

**Comment by pkra** [comment]
> For the physics.js, do you mean there are a lot of tab characters there?

Sorry for being incomplete earlier. I meant that the code seems commented out.

**Comment by ickc** [comment]
I added a few tests on my github pages. It seems fine. Since the comment are closed `\*...*\`, it shouldn't comment out the following codes.

**Comment by pkra** [comment]
D'oh -- my apologies! I hadn't looked carefully and was thrown off by GitHub's preview graying everything out. Thanks again for the great work!

**Comment by pkra** [comment]
https://cdn.mathjax.org/mathjax/contrib/physics/physics.js and https://cdn.mathjax.org/mathjax/contrib/physics/unpacked/physics.js should be up-to-date now (might need clearing the browser cache)

**Comment by pkra** [comment]
Thanks again @ickc !

**Comment by ickc** [comment]
Thank you!

**Pull request merged**